### PR TITLE
Fix table styling in surv analysis and drug tables

### PR DESCRIPTION
**Pull request by SaraKuitunen**
Fix the headers of survival analysis and drug tables: make background
be even and widgets be on the same level with each other (previously
different parts were not inline)

Allow HR results to wrap to make columns be inline

Change column widths to make p-values fit better

### CODE REVIEW

**Review by vincent-octo — COMMENTED** [review]
Hei @SaraKuitunen this is great, thanks a lot for this pull request!

I made few comments about minor revisions, let me know what you think about them. Otherwise it's ready to merge :raised_hands: 

Also, an even more minor comment: you could add "GitHub: closes #101" as a new line at the end of your commit message and it will automatically close the corresponding issue (see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)).

File: risteys_elixir/assets/js/AssocTable.vue [hunk 1]
```diff
@@ -1,248 +1,269 @@
...
-			<colgroup span="4"></colgroup>
-			<colgroup span="4"></colgroup>
-			<thead>
-				<tr>
-					<th rowspan="2">
-						<div class="border-t border-b h-full pl-2 pt-4">  <!-- div hack to have the borders stay in place on scroll -->
-							Endpoint<br>
-							<input
+	<div role="table" class="assoc-scrolling">
+		<div role="rowgroup" id="grid-assoc-header">
```
[the remark this answers is not in the document]
Comment (SaraKuitunen) [reply risteys_elixir/assets/js/AssocTable.vue:3]: Not really, now that I checked it again, there's no need for a grid-assoc-header.

File: risteys_elixir/assets/js/AssocTable.vue [hunk 2]
```diff
@@ -420,73 +441,63 @@ export default {
...
 
-/* thead: endpoint cell */
-.scrolling thead tr:first-child th:first-child {
-	left: 0;
-	z-index: 1;
-	height: 100px;
-}
+/* Grid table layout */
+#grid-assoc-header {
+	display: grid;
```
[the remark this answers is not in the document]
Comment (SaraKuitunen) [reply risteys_elixir/assets/js/AssocTable.vue:464]: Yes, you're right

File: risteys_elixir/assets/js/AssocTable.vue [hunk 3]
```diff
@@ -1,248 +1,269 @@
...
+		</div>
+
+		<div v-for="(pheno, idx) in assoc_table" role="rowgroup">
+			<!-- LAG: no lag -->
+			<div
+				role="row"
+				v-bind:class="bg_even(idx) + ' grid-assoc-body'"
+			>
+				<!-- ENDPOINT NAME -->
+				<div role="cell"> <!--v-bind:class="bg_even(idx)" ?? -->
```
[the remark this answers is not in the document]
Comment (SaraKuitunen) [reply risteys_elixir/assets/js/AssocTable.vue:96]: I guess I had initially, but eventually, I just forgot to remove the comment

**Review by vincent-octo — COMMENTED** [review]
Looks good to me! Ready to merge =)

**Pull request merged**